packages/mediapipe-task-text/lib/universal_mediapipe_text.dart

```diff
@@ -16,6 +17,9 @@ class TextClassifier extends BaseTextClassifier {
   @override
   Future<TextClassifierResult> classify(String text) =>
       throw UnimplementedError();
+
+  /// Cleans up all resources.
+  void dispose() {}
```

Comment:
Here and below, why the empty dispose methods?

Reply:
Oh my - I meant to put these on the abstract class. Oy vey!